lightning/src/ln/channelmanager.rs

```diff
@@ -4584,12 +4587,6 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 	/// The [`PaymentHash`] (and corresponding [`PaymentPreimage`]) must be globally unique. This
 	/// method may return an Err if another payment with the same payment_hash is still pending.
 	///
-	/// `user_payment_id` will be provided back in [`PaymentPurpose::InvoicePayment::user_payment_id`] events to
-	/// allow tracking of which events correspond with which calls to this and
-	/// [`create_inbound_payment`]. `user_payment_id` has no meaning inside of LDK, it is simply
-	/// copied to events and otherwise ignored. It may be used to correlate PaymentReceived events
-	/// with invoice metadata stored elsewhere.
-	///
```

Comment:
Would it be correct to say that `payment_secret` will be used for this after the follow-up?

Reply:
Ah, yeah I guess we won't duplicate those